assets/js/components/ClusterDetails/index.js

```diff
@@ -7,3 +7,5 @@ export { ClusterInfoBox } from './ClusterInfoBox';
 
 export default ClusterDetails;
 export { ClusterDetailsNew } from './ClusterDetailsNew';
+export { ClusterSettingsNew } from './ClusterSettingsNew';
+export { truncatedClusterNameClasses } from './ClusterDetails';
```

Comment:
Why do we need to export the `truncatedClusterNameClasses` object?
We were not exporting before, and it is not used anywhere else outside of this component